src/file/file.rpc.ts

```diff
@@ -28,9 +34,14 @@ const file_proto = protoDescriptor.file;
  */
 export class RPC {
```

Comment:
Can you change the name to be more indicative, like `FileService` or just `FILE` (because then you could use it as file.server and `FileServer.server`  would be weird).